radiotherapyai/metrics/dice.py

```diff
@@ -26,9 +29,48 @@ def from_shapely(
     Explanation of the Dice is available at:
     <https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient>
 
-    Args:
-        a (shapely.geometry.base.BaseGeometry)
-        b (shapely.geometry.base.BaseGeometry)
+    Parameters
+    ----------
+    a : shapely.geometry.base.BaseGeometry
+    b : shapely.geometry.base.BaseGeometry
+
+    Returns
+    -------
+    float
+        The Dice score
```

Comment:
it wouldn't hurt to identify the expected limits [0,1] and a ten words or left description of what it means (overlap).
Links are good for someone who would like more detail.
But most people are already hip deep in code, and if they are, wandering off for a deep dive to figure out "what is this function" is kind of a task...

Reply:
Yup, good call. Will fix.

Reply:
> I'll fix before merge.

Actually, would you prefer I merge this and then make those changes in a new PR so that the new changes are easier to review?

Or is it much of a muchness, and best to have it in this PR and then re-request your approval?

Reply:
New pr please